src/ignore.c

```diff
@@ -203,7 +203,7 @@ static int parse_ignore_file(
 			break;
 		}
 
-		match->flags = GIT_ATTR_FNMATCH_ALLOWSPACE | GIT_ATTR_FNMATCH_ALLOWNEG;
+		match->flags = GIT_ATTR_FNMATCH_ALLOWNEG | GIT_ATTR_FNMATCH_ALLOWSPACE;
```

Comment:
Is there a reason these need to change order?

Reply:
Nope, sorry.